internal/provider/data_source_schema_version.go

```diff
@@ -99,35 +76,44 @@ func (d schemaVersionDataSource) Read(ctx context.Context, req tfsdk.ReadDataSou
 		return
 	}
 
-	schema, err := d.provider.client.GetSchemaByName(data.FullName.Value)
+	axualSchema, err := d.provider.client.GetSchemaByName(data.FullName.ValueString())
```

Comment:
I'm guessing you renamed this to avoid conflict with terraform Schemas.

Reply:
Yes, I renamed this to avoid conflict with terraform Schemas.